python/pfs/drp/stella/createDetGeom.py

```diff
@@ -0,0 +1,108 @@
+import os
+import sys
+import argparse
+
+import lsst.pex.config                  as pexConfig
+import lsst.afw.table as afwTable
+import lsst.afw.geom as afwGeom
+from lsst.pipe.base import Task
+
+class CreateDetGeomConfig(pexConfig.Config):
+    """Configuration for creating the PFS detector geometry files"""
+
+class CreateDetGeomTask(Task):
+    """Task to create the detector geometry files for PFS"""
+    ConfigClass = CreateDetGeomConfig
+    _DefaultName = "createDetGeomTask"
+
+    def __init__(self, *args, **kwargs):
+        super(CreateDetGeomTask, self).__init__(*args, **kwargs)
+
+    def run(self):
+        gain = (1.24, 1.24, 1.27, 1.18, 1.26, 1.20, 1.24, 1.26)
+        rdnoise = (3.61, 3.78, 3.18, 2.95, 3.19, 3.80, 4.51, 3.18)
```

Comment:
It'd be nice to document where these come from. Maybe they're obvious to the veterans...